### PR TITLE
Update bay area circuits and gimpel logos

### PR DESCRIPTION
**Pull request by RyanThorna**
Updated sponsor page with Bay Area Circuits and Gimpel logos. Also removed or updated various others.

### CODE REVIEW

File: sponsors/index.html [hunk 1]
```diff
@@ -20,7 +20,7 @@ <h3 class="page-tree-text">
                 <div class="container mas-boxes">
                     <div class="mas-boxes-inner direc-hover-box animated flipInX">
                         <a target ="_blank" href="http://www.altium.com/">
-                            <img src="../assets/img/sponsor_logo/allcell.png" class="img-responsive" alt="">
+                            <img src="../assets/img/sponsor_logo/altium.jpg" class="img-responsive" alt="">
                             <div class="direc-overlay">
```
Comment (wheelerlaw) [inline sponsors/index.html]: This seems like a mistake, is it not?

File: sponsors/index.html [hunk 2]
```diff
@@ -96,11 +96,11 @@ <h3 class="page-tree-text">
...
             <div class="portfolio-masonary-wrapper">
                 <div class="container mas-boxes">
-                    <div class="mas-boxes-inner direc-hover-box wow animated flipInX">
-                        <a target ="_blank" href="http://www.ge.com/">
-                            <img src="../assets/img/sponsor_logo/ge.svg" class="img-responsive" alt="">
+                    <div class="mas-boxes-inner direc-hover-box animated flipInX">
+                        <a target ="_blank" href="https://bayareacircuits.com/">
+                            <img src="../assets/img/sponsor_logo/bayareacircuits.jpg" class="img-responsive" alt="">
                             <div class="direc-overlay">
-                                <span>General Electric</span>
```
Comment (wheelerlaw) [inline sponsors/index.html]: Is GE no longer a sponsor?

File: sponsors/index.html (hunk 1 above)
Comment (wheelerlaw) [inline sponsors/index.html]: Oh wait, nvm.

File: sponsors/index.html [hunk 3]
```diff
@@ -20,7 +20,7 @@ <h3 class="page-tree-text">
                 <div class="container mas-boxes">
                     <div class="mas-boxes-inner direc-hover-box animated flipInX">
                         <a target ="_blank" href="http://www.altium.com/">
-                            <img src="../assets/img/sponsor_logo/allcell.png" class="img-responsive" alt="">
+                            <img src="../assets/img/sponsor_logo/altium.jpg" class="img-responsive" alt="">
                             <div class="direc-overlay">
                                 <span>All Cell</span>
```
Comment (wheelerlaw) [inline sponsors/index.html]: This also needs to change to Altium.

**Comment by wheelerlaw** [comment]
Just one small change and I will approve the PR.

File: sponsors/index.html (hunk 2 above)
Comment (RyanThorna) [inline sponsors/index.html]: Correct

**Pull request merged**